src/db/partyDb.js

```diff
@@ -0,0 +1,14 @@
+const party = [
+    {
```

Comment:
two parties shouldn't have the same `id`, or what do you think?

Reply:
Yes i think so to.. Thank you moving forward i will adjust them